addons/website_sale/static/src/js/website_sale.js

```diff
@@ -129,6 +129,8 @@ var core = require('web.core');
 var config = require('web.config');
 var sAnimations = require('website.content.snippets.animation');
 require("website.content.zoomodoo");
+var wUtils = require('website_sale.utils');
+var qweb = core.qweb;
```

Comment:
Where do you use qweb ?

Reply:
(should have been removed when moved my code)